xbmc/XBDateTime.cpp

```diff
@@ -1327,12 +1327,12 @@ std::string CDateTime::GetAsLocalizedTime(const std::string &format, bool withSe
   return strOut;
 }
 
-std::string CDateTime::GetAsLocalizedDate(bool longDate/*=false*/) const
+std::string CDateTime::GetAsLocalizedDate(bool longDate/*=false*/, bool withYear /* = true */) const
 {
-  return GetAsLocalizedDate(g_langInfo.GetDateFormat(longDate));
+  return GetAsLocalizedDate(g_langInfo.GetDateFormat(longDate), withYear);
 }
 
-std::string CDateTime::GetAsLocalizedDate(const std::string &strFormat) const
+std::string CDateTime::GetAsLocalizedDate(const std::string &strFormat, bool withYear /* = true */) const
```

Comment:
why do we need this here? you should be able to leave the year out of the format pattern.